src/retrieve-data-helpers/smart-authorize.js

```diff
@@ -0,0 +1,6 @@
+import { localhostClientId as clientId } from '../config/client-id';
+
+FHIR.oauth2.authorize({
+  client_id: clientId,
+  scope: 'patient/*.* user/*.* launch openid profile online_access',
```

Comment:
Can scope be an argument or import here?  Depending on each implementation of FHIR and authorization server, wild card scope may/may not be acceptable.

Reply:
We can import it if we want to. 
https://github.com/cds-hooks/sandbox-2.0/pull/2/commits/edd5c152ce616fb85dd02eea0014873ced111905